neo/SmartContract/InteropService.cs

```diff
@@ -201,6 +202,22 @@ private static bool Runtime_Serialize(ApplicationEngine engine)
             return true;
         }
 
+        private static bool Runtime_GetNotifications(ApplicationEngine engine)
+        {
+            var data = engine.CurrentContext.EvaluationStack.Pop().GetByteArray();
+            if (data.Length != UInt160.Length) return false;
+            if (!engine.CheckArraySize(engine.Notifications.Count)) return false;
+
+            var hash = new UInt160(data);
+            engine.CurrentContext.EvaluationStack.Push
+                (
+                new VM.Types.Array(engine.Notifications
+                .Where(u => hash == UInt160.Zero || u.ScriptHash == hash)
```

Comment:
`hash == UInt160.Zero` What is this?

Reply:
Is for retrive all notifications, without filter